runtime/src/main/java/io/quarkus/tika/runtime/graal/TikaFeature.java

```diff
@@ -6,9 +6,17 @@
 public class TikaFeature implements Feature {
     @Override
     public void afterRegistration(AfterRegistrationAccess access) {
-        final String reason = "Quarkus run time init for Apache Tika";
-        RuntimeClassInitialization.initializeAtRunTime("org.apache.pdfbox.pdmodel", reason);
```

Comment:
Passing a reason is no longer supported, the reason was actually passed as a package name :)